tests/baseline/tsfresh_features_test.py

```diff
@@ -0,0 +1,417 @@
+import sys
+import os
+from time import time, sleep
+import shutil
+import datetime
+import csv
+import json
+import errno
+import tempfile
+
+import unittest
+from mock import Mock, patch
+import numpy as np
+import pandas as pd
+from tsfresh import extract_features, extract_relevant_features, select_features
+from tsfresh.examples.test_tsfresh_baseline_dataset import download_json_dataset
+
+from tsfresh_feature_names import TSFRESH_FEATURES, TSFRESH_BASELINE_VERSION
+
+python_version = int(sys.version_info[0])
+baseline_dir = os.path.dirname(os.path.realpath(__file__))
+tests_dir = os.path.dirname(baseline_dir)
+tsfresh_dir = os.path.dirname(tests_dir)
+baseline_ts_json_file = 'data.json'
+baseline_ts_json_baseline = '%s/tsfresh-%s.py%s.%s.features.transposed.csv' % (
+    baseline_dir, TSFRESH_BASELINE_VERSION, str(python_version),
+    baseline_ts_json_file)
+t_fname_out_fail = '%s/tsfresh/examples/data/test_tsfresh_baseline_dataset/tsfresh-unknown-version.py%s.data.json.features.transposed.csv' % (tsfresh_dir, str(python_version))
+test_path = tempfile.mkdtemp()
+fname_in = '%s/%s' % (test_path, baseline_ts_json_file)
+tmp_csv = '%s.tmp.csv' % (fname_in)
+fname_out = '%s.features.csv' % fname_in
+t_fname_out = '%s.features.transposed.csv' % fname_in
+
+
+class TestTsfreshBaseline(unittest.TestCase):
+    """
+    Test all the features and their calculated values with a 60 data point
+    sample of simple anomalous timeseries data set and compare that the feature
+    names and calculated values match the baselines calcualated for the specific
+    verison of tsfresh as defined by :mod:`tsfresh_feature_names.TSFRESH_FEATURES`
```

Comment:
@earthgecko Normally if I do a review, I add the review to the line I am referring to. As the line is not so long, I thought it would be easy enough to spot. But, here it is clearer:
verison -> version